group.yml

```diff
@@ -518,3 +518,196 @@ external_scanners:
       # This flag enables for this OCP version, but it also has to be enabled specifically in the image config
       # as well
       enabled: true
+
+      # Components that ART is shipping but not building
```

Comment:
Can you also describe why/how we need this and we're planning to use

Reply:
This is just for us to keep track of these components. If we see any component besides this in the brew tags, OCPBUGS pipeline will complain so that we can check.

Reply:
At Justin's [suggestion](https://redhat-internal.slack.com/archives/GDBRP5YJH/p1700597004078439?thread_ts=1700547510.892639&cid=GDBRP5YJH)